### DISCUSSION TITLE
Dyamic interface field/notice based on others field

### DISCUSSION THREAD

**Discussion by laekem34** [directus/directus#10364] [Feature Requests]
Hello,

I don't know if it's already possible, but it could be great if we could have fields or dynamic notice in the interface according to the value of another fields, and refreshed in front according the fields values.
I take as an example the map where we cannot directly read the gps coordinates.
We could add a remark field with inside the possibility of calling the item itself with `$ITEM`.
This variable for also applies to a hidden field for example to calculate a value according to other columns.

![image](https://user-images.githubusercontent.com/21008981/144992868-ba6e3884-da2c-4c18-b9f4-ce096c495ee7.png)
![image](https://user-images.githubusercontent.com/21008981/144993010-d08c34f8-023c-41c2-a270-db3a34df16f0.png)

Thanks you for making that amazing tool !

**Comment by rijkvanzanten** [comment]
Love that idea!

I'm wondering if we can somehow integrate this with another idea I had that allows you to put translated values in any input like this (through a similar syntax: `$t:my-key`). Those two combined would allow for some crazy powerful configurations :o (cc @benhaynes)

**Reply by laekem34** [reply]
Yes !! that's an amazing idea !

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
